externalLibrary/src/Encoders.elm

```diff
@@ -0,0 +1,116 @@
+module Encoders exposing (model, products)
```

Comment:
Since you asked about general Elm style comments, I usually like to put a type and it's encoders and decoders in a a single file (along with any type-specific helper functions).  Generally, if I am going to change the type, I need to change the encoder and decoder as well so having them in the same file is pretty convenient. Also, using the decoders and encoders reads nicely when written as something like `Item.decoder`

For this project, I would probably also split out the types and encoders and decoders that are based on the Plaid interface and keep them in a distinct namespace from models that define how the app works. You could put them all in a big `Plaid` module, or split them out into things like `Plaid.Account, etc. 

These are just my personal preferences though. YMMV